kubernetes/zenko/charts/backbeat/templates/ingestion/producer_deployment.yaml

```diff
@@ -46,6 +46,8 @@ spec:
               value: "{{ template "backbeat.redis-hosts" . }}"
             - name: REDIS_HA_NAME
               value: "{{ .Values.redis.sentinel.name }}"
+            - name: QUEUE_POPULATOR_BATCH_MAX_READ
+              value: "1000"
```

Comment:
https://github.com/scality/Zenko/blob/eac712e601bbf9fbe6c78e1706724f14f2e78ee4/kubernetes/zenko/charts/backbeat/values.yaml#L60 Maybe it's preferred that this value be added there, so it's easier for tuning (all values and variables in one file)

Reply:
I'm sort of conflicted with this change itself since I feel like it shouldn't be tuneable by a user. I'd rather try and avoid more flexibility for adjusting this value

Reply:
If that's the case, shouldn't we leave it out of the deployment chart in general, and only reference/set this value within the backbeat code?

Reply:
Does it actually affect CRR too?

Reply:
only if adding to the replication pod charts